### ISSUE TITLE
mean age text below figure

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [iantaylor-NOAA/Lingcod_2021#125]
Text that was echoed is printing below figure.

**Comment by kellijohnson-NOAA** [comment]
@iantaylor-NOAA I am not seeing this any more are you? If it is gone from your builds too then we can close this issue.

**Comment by iantaylor-NOAA** [comment]
I'm still seeing this one instance: 
![image](https://user-images.githubusercontent.com/4992918/127394032-d8869098-92d1-4eda-b9ff-18a9c624f2ce.png)

**Comment by kellijohnson-NOAA** [comment]
Thanks Ian, so I am. I was looking at the South document. I will get that fixed.

**Comment by iantaylor-NOAA** [comment]
Maybe the problem is this stuff in the caption of the mean age figs:
```
<br><br>For more info, see<br> <blockquote>Francis, R.I.C.C. (2011). Data weighting in statistical fisheries stock assessment models. <i>Can. J. Fish. Aquat. Sci.</i> 68: 1124-1138.</blockquote>
```
We don't reference those figs and nobody has seen them in the document yet, so we could just 
replace
```
add_figure_ling(csv = figurecsv, "comp_condAALfit")`
```
in `53figures.Rmd` with 
```
add_figure_ling(csv = figurecsv, ""comp_condAALfit_resids")
add_figure_ling(csv = figurecsv, ""comp_condAALfit_Andre_plots")
```

**Comment by iantaylor-NOAA** [comment]
I went ahead and made the fix proposed above in commit cf3667d576fdec533026fb386261c793064100ef which worked successfully.
